docs/overrides/main.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block announce %}
+  <a href="https://discord.com/invite/7rfW4kg6B5" class="cli-announcement">
```

Comment:
Should we add the server URL as well as the invite URL, to make it easy for people who are already a part of the server to open it?